lib/bap_image/sectionRef.mli

```diff
@@ -0,0 +1,38 @@
+(** Signle section in the list of sections int the object file *)
```

Comment:
Maybe it possible to call Or_error create from string function from c code? Idea is that allmost llvm methods returns error_code object, which can be easely converted to string.